VideoLocker/src/main/java/org/edx/mobile/module/db/impl/IDatabaseImpl.java

```diff
@@ -32,7 +33,9 @@ public IDatabaseImpl(Context context, LoginPrefs loginPrefs) {
 
     @Nullable
     private String username() {
-        return loginPrefs.getUsername();
+        final String username = loginPrefs.getUsername();
+        if (username != null) return Sha1Util.SHA1(username);
```

Comment:
nit pick: Can we make this one return statement?  Something like:
return (username != null) ? Sha1Util.SHA1(username) : null;